===================================================================
RCS file: /cvsroot/incrtcl/incrTcl/itcl/tests/ensemble.test,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- tests/ensemble.test	2000/06/01 20:34:35	1.3
+++ tests/ensemble.test	2001/11/05 19:42:11	1.4
@@ -6,7 +6,7 @@
 #            mmclennan@lucent.com
 #            http://www.tcltk.com/itcl
 #
-#      RCS:  $Id: ensemble.test,v 1.3 2000/06/01 20:34:35 wart Exp $
+#      RCS:  $Id: ensemble.test,v 1.4 2001/11/05 19:42:11 hobbs Exp $
 # ----------------------------------------------------------------------
 #            Copyright (c) 1993-1998  Lucent Technologies, Inc.
 # ======================================================================
@@ -45,8 +45,13 @@
 } {{one: 1} {two: 2 3} {three: 3 4 5}}
 
 test ensemble-1.5 {invoking parts with improper arguments} {
-    list [catch "test_numbers three x" msg] $msg
-} {1 {no value given for parameter "y" to "test_numbers three"}}
+    set res [catch "test_numbers three x" msg]
+    if {[package vsatisfies [package provide Tcl] 8.4]} {
+	lappend res [string match "wrong # args*" $msg]
+    } else {
+	lappend res [string match "no value given*" $msg]
+    }
+} {1 1}
 
 test ensemble-1.6 {errors trigger a usage summary} {
     list [catch "test_numbers foo x y" msg] $msg
